### ISSUE TITLE
[Bug]: 我节点可以测速，但是选中之后IP依然是实际IP，无法科学上网。系统代理也开了，端口设置正常，手机上节点可以使用，其他电脑也可以使用。

### ISSUE THREAD

**Issue by xingchen-yan** [Fndroid/clash_for_windows_pkg#4274]
### 请认真检查以下清单中的每一项

- [ ] 已经搜索过，没有发现类似issue
- [X] 已经搜索过文档，没有发现相关内容
- [X] 已经尝试使用过最新版，问题依旧存在
- [X] 使用的是官方版本（未替换及修改过安装目录程序文件）

### 软件版本

0.20.19

### 操作系统

Windows x64

### 系统版本

win10

### 问题描述

我节点可以测速，但是选中之后IP依然是实际IP，无法科学上网。系统代理也开了，端口设置正常，手机上节点可以使用，其他电脑也可以使用。
![1](https://user-images.githubusercontent.com/76110297/230258331-be24d1ee-467e-45f1-a86e-afa9c39c54a8.png)
![image](https://user-images.githubusercontent.com/76110297/230258385-b14ec0ce-fbaf-42a4-a91e-babf00b0bff5.png)
![image](https://user-images.githubusercontent.com/76110297/230258426-5d0aadc0-1888-4e01-ac1a-026b8f511696.png)

### 复现步骤

我打开clash后进行节点测速，测速后进行选择节点，选择节点后测试IP仍在国内.

### 日志文件

<details><summary>logs</summary><p>

```
10:18:49 INF [API] listening addr=127.0.0.1:51186
10:18:49 INF [Inbound] Mixed(http+socks) listening addr=127.0.0.1:7890
10:18:49 WRN [TCP] dial failed error=lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. proxy=DIRECT lAddr=127.0.0.1:51198 rAddr=raw.githubusercontent.com:443
10:18:49 WRN [TCP] dial failed error=lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. proxy=DIRECT lAddr=127.0.0.1:51201 rAddr=raw.githubusercontent.com:443
10:18:49 INF [TCP] connected, doesn't match any rule lAddr=127.0.0.1:51199 rAddr=api.github.com:443 proxy=DIRECT
10:18:49 WRN [TCP] dial failed error=lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. proxy=DIRECT lAddr=127.0.0.1:51196 rAddr=raw.githubusercontent.com:443
10:18:49 WRN [TCP] dial failed error=lookup raw.githubusercontent.com: getaddrinfow: The requested name is valid, but no data of the requested type was found. proxy=DIRECT lAddr=127.0.0.1:51207 rAddr=raw.githubusercontent.com:443
10:18:49 INF [TCP] connected, doesn't match any rule lAddr=127.0.0.1:51197 rAddr=api.github.com:443 proxy=DIRECT
10:19:25 INF [TCP] connected, doesn't match any rule lAddr=127.0.0.1:51219 rAddr=charlie.yns5ub5cr1b3.vip:443 proxy=DIRECT
10:19:27 INF [Config] initial compatible provider name=自动优选
10:19:27 INF [Config] initial compatible provider name=国外流量
10:20:08 INF [TCP] connected lAddr=127.0.0.1:51263 rAddr=array513.prod.do.dsp.mp.microsoft.com:443 mode=global
10:20:21 INF [TCP] connected lAddr=127.0.0.1:51297 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:20:25 INF [TCP] connected lAddr=127.0.0.1:51303 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:20:33 INF [TCP] connected lAddr=127.0.0.1:51309 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:21:05 INF [TCP] connected lAddr=127.0.0.1:51321 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:22:07 INF [TCP] connected lAddr=127.0.0.1:51336 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:22:10 INF [TCP] connected lAddr=127.0.0.1:51345 rAddr=self.events.data.microsoft.com:443 mode=global
10:22:56 INF [TCP] connected lAddr=127.0.0.1:51355 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:23:04 INF [TCP] connected lAddr=127.0.0.1:51360 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:24:37 INF [TCP] connected lAddr=127.0.0.1:51391 rAddr=self.events.data.microsoft.com:443 mode=global
10:24:39 INF [TCP] connected lAddr=127.0.0.1:51394 rAddr=www.bing.com:443 mode=global
10:24:39 INF [TCP] connected lAddr=127.0.0.1:51396 rAddr=www.bing.com:443 mode=global
10:25:52 INF [TCP] connected lAddr=127.0.0.1:51402 rAddr=g.live.com:443 mode=global
10:25:52 INF [TCP] connected lAddr=127.0.0.1:51404 rAddr=oneclient.sfx.ms:443 mode=global
10:25:53 INF [TCP] connected lAddr=127.0.0.1:51406 rAddr=self.events.data.microsoft.com:443 mode=global
10:25:53 INF [TCP] connected lAddr=127.0.0.1:51409 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:25:53 INF [TCP] connected lAddr=127.0.0.1:51413 rAddr=config.teams.microsoft.com:443 mode=global
10:26:17 INF [TCP] connected lAddr=127.0.0.1:51423 rAddr=greenhubtx.ga:443 mode=global
10:26:17 INF [TCP] connected lAddr=127.0.0.1:51426 rAddr=edge.microsoft.com:443 mode=global
10:26:17 INF [TCP] connected lAddr=127.0.0.1:51428 rAddr=ajax.aspnetcdn.com:443 mode=global
10:26:17 INF [TCP] connected lAddr=127.0.0.1:51430 rAddr=cdn.jsdelivr.net:443 mode=global
10:26:17 INF [TCP] connected lAddr=127.0.0.1:51432 rAddr=cdnjs.cloudflare.com:443 mode=global
10:26:22 INF [TCP] connected lAddr=127.0.0.1:51435 rAddr=cn.bing.com:443 mode=global
10:26:24 INF [TCP] connected lAddr=127.0.0.1:51438 rAddr=github.githubassets.com:443 mode=global
10:26:24 INF [TCP] connected lAddr=127.0.0.1:51437 rAddr=github.com:443 mode=global
10:26:38 INF [TCP] connected lAddr=127.0.0.1:51441 rAddr=v10.events.data.microsoft.com:443 mode=global
10:26:55 INF [TCP] connected lAddr=127.0.0.1:51445 rAddr=github.com:443 mode=global
10:26:56 INF [TCP] connected lAddr=127.0.0.1:51447 rAddr=self.events.data.microsoft.com:443 mode=global
10:26:57 INF [TCP] connected lAddr=127.0.0.1:51449 rAddr=github.githubassets.com:443 mode=global
10:26:57 INF [TCP] connected lAddr=127.0.0.1:51450 rAddr=avatars.githubusercontent.com:443 mode=global
10:27:02 INF [TCP] connected lAddr=127.0.0.1:51453 rAddr=api-edge.cognitive.microsofttranslator.com:443 mode=global
10:27:41 INF [TCP] connected lAddr=127.0.0.1:51456 rAddr=v20.events.data.microsoft.com:443 mode=global
10:27:57 INF [TCP] connected lAddr=127.0.0.1:51458 rAddr=github.githubassets.com:443 mode=global
10:28:24 INF [TCP] connected lAddr=127.0.0.1:51462 rAddr=avatars.githubusercontent.com:443 mode=global
10:28:24 INF [TCP] connected lAddr=127.0.0.1:51460 rAddr=github.githubassets.com:443 mode=global
10:28:28 INF [TCP] connected lAddr=127.0.0.1:51465 rAddr=cn.bing.com:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51479 rAddr=sdk.51.la:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51470 rAddr=jspassport.ssl.qhimg.com:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51480 rAddr=s.ssl.qhres2.com:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51469 rAddr=www.ip.cn:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51475 rAddr=js.users.51.la:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51482 rAddr=npm.elemecdn.com:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51476 rAddr=cdn.bootcss.com:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51473 rAddr=s9.cnzz.com:443 mode=global
10:28:30 INF [TCP] connected lAddr=127.0.0.1:51467 rAddr=ip.cn:443 mode=global
10:28:31 INF [TCP] connected lAddr=127.0.0.1:51485 rAddr=www.layuicdn.com:443 mode=global
10:28:32 INF [TCP] connected lAddr=127.0.0.1:51487 rAddr=www.layuicdn.com:443 mode=global
10:29:25 INF [TCP] connected lAddr=127.0.0.1:51491 rAddr=github.githubassets.com:443 mode=global
10:29:42 INF [TCP] connected lAddr=127.0.0.1:51493 rAddr=github.githubassets.com:443 mode=global
10:30:44 INF [TCP] connected lAddr=127.0.0.1:51498 rAddr=cn.bing.com:443 mode=global
10:31:25 INF [TCP] connected lAddr=127.0.0.1:51504 rAddr=github.githubassets.com:443 mode=global
10:31:25 INF [TCP] connected lAddr=127.0.0.1:51503 rAddr=github.com:443 mode=global
10:31:41 INF [TCP] connected lAddr=127.0.0.1:51508 rAddr=api-edge.cognitive.microsofttranslator.com:443 mode=global
10:31:50 INF [TCP] connected lAddr=127.0.0.1:51510 rAddr=activity.windows.com:443 mode=global
10:32:10 INF [TCP] connected lAddr=127.0.0.1:51515 rAddr=self.events.data.microsoft.com:443 mode=global
10:32:12 INF [TCP] connected lAddr=127.0.0.1:51517 rAddr=tile-service.weather.microsoft.com:80 mode=global
10:32:15 INF [TCP] connected lAddr=127.0.0.1:51519 rAddr=github.com:443 mode=global
10:32:44 INF [TCP] connected lAddr=127.0.0.1:51522 rAddr=cn.bing.com:443 mode=global
10:32:44 INF [TCP] connected lAddr=127.0.0.1:51524 rAddr=v20.events.data.microsoft.com:443 mode=global
10:33:05 INF [TCP] connected lAddr=127.0.0.1:51528 rAddr=github.githubassets.com:443 mode=global
10:33:05 INF [TCP] connected lAddr=127.0.0.1:51527 rAddr=github.com:443 mode=global
10:33:40 INF [TCP] connected lAddr=127.0.0.1:51532 rAddr=github.com:443 mode=global
10:33:40 INF [TCP] connected lAddr=127.0.0.1:51534 rAddr=nav-edge.smartscreen.microsoft.com:443 mode=global
10:33:42 INF [TCP] connected lAddr=127.0.0.1:51536 rAddr=github.githubassets.com:443 mode=global
10:33:42 INF [TCP] connected lAddr=127.0.0.1:51538 rAddr=avatars.githubusercontent.com:443 mode=global
10:33:42 INF [TCP] connected lAddr=127.0.0.1:51540 rAddr=api-edge.cognitive.microsofttranslator.com:443 mode=global
10:34:03 INF [TCP] connected lAddr=127.0.0.1:51542 rAddr=activity.windows.com:443 mode=global
10:34:19 INF [TCP] connected lAddr=127.0.0.1:51546 rAddr=github.githubassets.com:443 mode=global
10:34:28 INF [TCP] connected lAddr=127.0.0.1:51550 rAddr=sb.adtidy.org:443 mode=global
10:35:06 INF [TCP] connected lAddr=127.0.0.1:51553 rAddr=activity.windows.com:443 mode=global
10:35:44 INF [TCP] connected lAddr=127.0.0.1:51558 rAddr=cn.bing.com:443 mode=global
10:35:45 INF [TCP] connected lAddr=127.0.0.1:51561 rAddr=github.githubassets.com:443 mode=global
10:35:45 INF [TCP] connected lAddr=127.0.0.1:51560 rAddr=github.com:443 mode=global
10:36:27 INF [TCP] connected lAddr=127.0.0.1:51566 rAddr=github.com:443 mode=global
10:36:28 INF [TCP] connected lAddr=127.0.0.1:51568 rAddr=github-production-user-asset-6210df.s3.amazonaws.com:443 mode=global
10:36:41 INF [TCP] connected lAddr=127.0.0.1:51573 rAddr=github-production-user-asset-6210df.s3.amazonaws.com:443 mode=global
10:37:00 INF [TCP] connected lAddr=127.0.0.1:51578 rAddr=github-production-user-asset-6210df.s3.amazonaws.com:443 mode=global
10:37:12 INF [TCP] connected lAddr=127.0.0.1:51580 rAddr=array513.prod.do.dsp.mp.microsoft.com:443 mode=global
10:37:21 INF [TCP] connected lAddr=127.0.0.1:51584 rAddr=github-production-user-asset-6210df.s3.amazonaws.com:443 mode=global
10:37:30 INF [TCP] connected lAddr=127.0.0.1:51587 rAddr=self.events.data.microsoft.com:443 mode=global
10:37:32 INF [TCP] connected lAddr=127.0.0.1:51590 rAddr=www.bing.com:443 mode=global
10:37:40 INF [TCP] connected lAddr=127.0.0.1:51594 rAddr=github-production-user-asset-6210df.s3.amazonaws.com:443 mode=global
10:37:46 INF [TCP] connected lAddr=127.0.0.1:51596 rAddr=github.githubassets.com:443 mode=global
10:38:03 INF [TCP] connected lAddr=127.0.0.1:51599 rAddr=edge.microsoft.com:443 mode=global
10:39:06 INF [TCP] connected lAddr=127.0.0.1:51603 rAddr=objects-origin.githubusercontent.com:443 mode=global
```

</p></details>

### 其他补充

![image](https://user-images.githubusercontent.com/76110297/230258815-064677b5-97df-4888-abfe-69e0b64a1956.png)

**Comment by xingchen-yan** [comment]
emmm
突然能用了又 
不明白为什么

**Comment by xingchen-yan** [comment]
突然可以使用了 不清楚为什么

**Comment by Fndroid** [comment]
看看你配置文件里是不是有dns的设置，有就删掉

**Comment by HelloWorldJie02** [comment]
我也和你是同样问题，请问现在知道是怎么解决的了吗？

**Comment by Fndroid** [comment]
This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 7 days
